websites/Y/YomuMangás/presence.ts

```diff
@@ -1,106 +1,84 @@
-			"/faq": { details: "Lendo o FAQ" },
-			"/tos": { details: "Lendo os Termos de Uso" },
-			"/privacy": { details: "Lendo as Políticas de Privacidade" },
-			"/dmca": { details: "Lendo DMCA" },
-			"/search": { details: "Fazendo uma Busca Avançada" },
-			"/about": { details: "Lendo Sobre a Yomu" },
-			"/contact": { details: "Página de Contato" },
-			"/donate": { details: "Página de Doação" },
-			"/donators": { details: "Lista de Doadores" },
-			"/partners": { details: "Lista de Parceiros" },
-			"/staffs": { details: "Lista da Equipe" },
-			"/ranks": { details: "Página de Ranks" },
-			"/settings": { details: "Configurações" },
-			"/minigames": { details: "Página de Minigames" },
+			"/404": { details: "404", state: "Uhn? Onde estamos?" },
+			"/500": { details: "500", state: "Uhn? Algo deu errado..." },
+			"/search": { details: "Pesquisa Avançada" },
+			"/donate": { details: "Página de Doações" },
+			"/help": { details: "Página de Ajuda" },
+			"/tutorial": { details: "Página de Tutorial" },
+			"/shop": { details: "Loja", state: "Gastando Pães de Mel" },
 		};
 
-	if (privacy) {
-		presenceData.details = "Criando um novo mundo!";
-		presenceData.state = "yomumangas.com";
-	} else if (pathname.includes("/manga")) {
-		const isReader = pathname.includes("/chapter"),
-			isAdult = document.querySelector("#premid-adult")?.textContent === "true",
-			mangaTitle = document.querySelector("#premid-manga-title")?.textContent,
-			chapterTitle = document.querySelector(
-				"#premid-chapter-title"
-			)?.textContent;
-
-		if (images && !isAdult) {
-			presenceData.largeImageKey = document.querySelector<HTMLAnchorElement>(
-				"#premid-manga-image"
-			)?.href;
-		}
-		if (!manga || (isAdult && !adult))
-			presenceData.details = isReader ? "Lendo um Mangá" : "Página de um Mangá";
-		if (manga && ((isAdult && adult) || !isAdult)) {
-			presenceData.details = isReader
-				? `Lendo ${mangaTitle}`
-				: `Página de ${mangaTitle}`;
-		}
-		if (chapter && isReader) {
-			presenceData.state = `Cap. ${
-				document.querySelector("#premid-chapter-number")?.textContent
-			}${chapterTitle ? ` - ${chapterTitle}` : ""}`;
-		}
-		if (manga && ((isAdult && adult) || !isAdult)) {
+	if (!privacy) {
+		if (pathname.startsWith("/manga")) {
+			const isChapter = pathname.includes("/chapter/"),
+				isNsfw =
+					document.querySelector("#premid-manga-nsfw")?.textContent === "true",
+				chapter =
+					document.querySelector("#premid-manga-chapter")?.textContent || "0",
+				image = document.querySelector<HTMLImageElement>(
+					"#premid-manga-cover"
+				)?.src;
+			if (!isNsfw || (isNsfw && nsfw)) {
+				presenceData.details =
+					document.querySelector("#premid-manga-title")?.textContent || "Obra";
+				presenceData.state = !isChapter
+					? "Perfil da Obra"
+					: isNaN(Number(chapter))
+					? chapter
+					: `Capítulo ${chapter}`;
+				presenceData.largeImageKey =
+					isNsfw || !image ? "https://i.imgur.com/usrs3S5.png" : image;
```

Comment:
please use a 512x512 image

Reply:
It's the standard currently being used by this repository to use 512x512 images. 512x512 generally has a good quality while being smaller in size compared to the larger 1024x1024

Reply:
> It's the standard currently being used by this repository to use 512x512 images. 512x512 generally has a good quality while being smaller in size compared to the larger 1024x1024

I understand, thanks